spec/features/user_logs_in_spec.rb

```diff
@@ -1,3 +1,5 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.